lib/google/gax/settings.rb

```diff
@@ -225,7 +227,7 @@ def construct_settings(service_name, client_config, config_overrides,
           bundler: construct_bundling(bundling_config,
                                       bundle_descriptor),
           bundle_descriptor: bundle_descriptor,
-          kwargs: kwargs
+          kwargs: kwargs, errors: errors
```

Comment:
Adding errors to the next line would cause a travis failure of having a function with more than 35 lines. How would you suggest I fix the stylistic mistake while avoiding the travis failure?